api/app/GraphQL/Mutations/MarkNotificationAsRead.php

```diff
@@ -22,7 +22,6 @@ public function __invoke($_, array $args)
 
         if (! is_null($notification)) {
             $notification->markAsRead();
-            User::enrichNotification($notification);
```

Comment:
Why delete this? I believe we should run this wherever we return a notification, no?

Reply:
See: https://github.com/GCTC-NTGC/gc-digital-talent/pull/9694#discussion_r1520347907